package.json

```diff
@@ -21,19 +21,19 @@
     "node": "^8.0.0"
```

Comment:
Shouldn't we change this to LTS as usually?

Reply:
Or remove it entirely, it's not present in our [stylelint order project](https://github.com/visionappscz/stylelint-config-visionapps-order/blob/feature/update-dependencies/package.json).